### DISCUSSION TITLE
New image only works after restarting the server

### DISCUSSION THREAD

**Discussion by sogeil** [ltsp/ltsp#833] [Q&A]
Hello!

After "sudo ltsp image /" the clients can't load (or use?) the new image. They appear to load, but end up with a black, blank screen. The clients can only use the new image after a complete restart of the server.

Also
ltsp image /
ltsp ipxe
ltsp nfs
ltsp initrd
change nothing about it.

What do I have to do so that the server does not have to be restarted to make the image usable?

Greeting,
Olaf

Server: bodhi Linux 6

**Comment by alkisg** [comment]
Hi, I haven't seen that issue in any other distribution so it sounds like something is borked with Bodhi.
If you want, test with [Bodhi 7.0 beta](https://sourceforge.net/projects/bodhilinux/files/7.0.0-beta/). If it happens there as well, I might install it in a VM and test it when I have some free time.

**Comment by sogeil** [comment]
Hello!
Okay, thank you. I have some more important other issues that I need to solve first. If that should work, I'll test it with bodhi 7.

Greeting,
Olaf
